bigchaindb/common/transaction.py

```diff
@@ -410,7 +410,7 @@ class Transaction(object):
     TRANSFER = 'TRANSFER'
     GENESIS = 'GENESIS'
     ALLOWED_OPERATIONS = (CREATE, TRANSFER, GENESIS)
-    VERSION = bigchaindb.version.__version__
+    VERSION = bigchaindb.version.__short_version__[:-4]  # 0.9, 0.10 etc
```

Comment:
Well, I think what @libscott is trying to say is that if the `[:4]` implementation ever breaks, the tests would catch it (and we can react accordingly).

Anyway, thinking about releases, this is likely to never be a problem since 1.0 will remove this bit of complexity.